shared/menubar/index.js

```diff
@@ -187,8 +187,8 @@ class Menubar extends Component {
 
 export default connect(
   state => ({
-    username: state.config && state.config.status && state.config.status.user && state.config.status.user.username,
-    loggedIn: state.config && state.config.status && state.config.status.loggedIn,
+    username: state.config.username,
```

Comment:
let's keep this simple. we might want to discuss how we sync up the store and all this in a different way so lets defer